### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by denzelchung**
- Update landing page: filters, category selection, navigation to detail page
- Add Google Analytics to website - to change tracking ID (UA-xxxxxxx-x) in PortalLayout.jsx file
- Remove component folder - shifted to components for consistency with layouts and pages.

**Pending**
- Confirmation of categories
- Confirmation of images on the right panel
- Responsive design for navbar

### CODE REVIEW

File: public/slider/range.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+	opacity: .5;
+}
+
+.ui.range.disabled .inner:hover {
+	cursor: auto;
+}
+
+.ui.range.disabled .inner .track-fill {
+	background: #ccc;
+}
```
Comment (OoiYueYing) [inline public/slider/range.css:184]: is this file needed?

File: public/slider/range.js [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+	step         : 1,
+	start        : 0,
+	input        : false,
+
+	onChange     : function(value){},
+
+};
+
+
+})( jQuery, window, document );
```
Comment (OoiYueYing) [inline public/slider/range.js:273]: is this file needed?

File: src/components/LandingFilter.jsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  font-size: 50px;
+  font-weight: bold;
+  color: ${theme.colours.maroon};
+  :hover {
+    color: ${theme.colours.maroon};
+    opacity: 0.8;
+  }
+`;
+
+const regionOptions = [
```
Comment (OoiYueYing) [inline src/components/LandingFilter.jsx:21]: can we extract these constant values into the config folder? like create a constant.js file with these constant values.

File: src/components/LandingFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+    rating: '',
+    price: '100'
+  });
+  const onChange = (e, result) => {
+    const { name, value } = result || e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  useEffect(() => {
+    $(document).ready(() => {
```
Comment (OoiYueYing) [inline src/components/LandingFilter.jsx:56]: i don think we neeed the $(document).ready here cos useEffect runs both after the first render and after every update

File: src/components/Timeline.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  outline: none;
+}
+.TimelineDot > span:hover {
+  color: black;
+  cursor: pointer;
+}
+.TimelineDot.active {
+  color: black;
+  box-shadow: 0 0 0 0.3em #960F4A inset;
+}
```
Comment (OoiYueYing) [inline src/components/Timeline.css:43]: let's group timeline.jsx and timeline.css into a folder so we have components/timeline

File: public/slider/range.css (hunk 1 above)
Comment (denzelchung) [reply public/slider/range.css:184]: yea, for the range slider in the main page

File: src/components/LandingFilter.jsx [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    rating: '',
+    price: '100'
+  });
+  const onChange = (e, result) => {
+    const { name, value } = result || e.target;
+    setValues({ ...values, [name]: value });
+  };
+
+  useEffect(() => {
+    $('#amount-slider').range({
```
Comment (ooihuiying) [inline src/components/LandingFilter.jsx:38]: Is it possible to not mix raw css with react? Perhaps there is a slider plugin we can import?

File: src/components/LandingFilter.jsx [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+import {
+  Dropdown, Grid, Form
+} from 'semantic-ui-react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import theme from '../config/theme';
+import { regionOptions, paxOptions, ratingOptions } from '../config/constant';
+
+const GoButton = styled(Link)`
+  font-size: 50px;
```
Comment (ooihuiying) [inline src/components/LandingFilter.jsx:13]: @OoiYueYing @denzelchung Lets refactor the files.

For each component, we should create 1 folder with 2 files inside.
For eg here, we can have a folder titled: LandingFilter. Inside LandingFilter folder, we can have LandingFilter.jsx and LandingFilter.styles.jsx to separate the code for logic and style.

File: src/components/LandingFilter.jsx [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+  return (
+    <Grid container>
+      <Grid.Column mobile={4} tablet={3} computer={2} />
+      <Grid.Column mobile={12} tablet={8} computer={6}>
+        <Form>
+          <Form.Field>
+            <h1 style={{ fontSize: '3rem', fontWeight: '500' }}>{category}</h1>
+            <div style={{ color: theme.colours.maroon, textAlign: 'justify' }}>
+              A powerful search engine that helps you compare pricing
+              from different venue rental websites
```
Comment (ooihuiying) [inline src/components/LandingFilter.jsx:62]: As discussed in the meeting, lets refactor the different form field terms into different components :)
